src/app/pages/subjects/new/import-subjects-listing.js

```diff
@@ -0,0 +1,3 @@
+import SubjectsListing from './subjects-listing';
+
+export default SubjectsListing;
```

Comment:
I'm not following why this is needed vs importing directly from subjects-listing, is there a circular dependency issue?

Reply:
I cannot dynamically import TS files, so I make a little JS importer for each TS file I need to import using `import()`. There are probably webpack plugins and whatnot that can make it import TS, but this was pretty easy.